app/views/compute_resources_vms/form/azurerm/_base.html.erb

```diff
@@ -183,8 +183,10 @@
                    :disabled   => images.empty?,
                    :label      => _('Image'),
                    :required   => true,
+                   :selected   => images.map { |i| i.name if i.id == @host.image_id }.find { |i| !i.nil? },
```

Comment:
Trying to set the image name but does not set, although I am getting the value.

Reply:
although this won't work for the first time loading of VM tab in host create page since `@host` will be `nil` at that time.